pallets/teerex/src/lib.rs

```diff
@@ -132,6 +132,10 @@ pub mod pallet {
 	#[pallet::getter(fn enclave_count)]
 	pub type EnclaveCount<T: Config> = StorageValue<_, u64, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn schedule_enclave)]
+	pub type ScheduleEnclave<T: Config> = StorageValue<_, bool, ValueQuery>;
```

Comment:
We should have a better name.

We have `ScheduledEnclave` already which can be easily mixed up with this one.

How about `ScheduledEnclaveCheck` or on the contrary, `SkipScheduledEnclaveCheck`? Please take the default value (`false`) into account - maybe the latter is better, as we don't want to skip this check by default in prod.

Reply:
Alternatively we should put it to genesis

Reply:
> How about ScheduledEnclaveCheck or on the contrary, SkipScheduledEnclaveCheck? Please take the default value (false) into account - maybe the latter is better, as we don't want to skip this check by default in prod.

I agree, I will update the code

Reply:
So I did the above and updated the scripts for `local-setup` and `deploy.sh`